src/elements/common/item/IconCell.js

```diff
@@ -1,78 +1,93 @@
-/**
- * @flow
- * @author Box
- */
+Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
this should have been converted to a js.flow file instead of trying to add the defineProperty